src/main/java/Commands/Command.java

```diff
@@ -0,0 +1,30 @@
+package Commands;
```

Comment:
Perhaps it would be better to standardize package names to be in fully lowercase
```suggestion
package commands;
```